### ISSUE TITLE
Training problem on lidar data only

### ISSUE THREAD

**Issue by ChristineRYY** [sshaoshuai/PointRCNN#60]
I use lidar data only to train PointRcnn, is the bin-based loss of axis xyz still applicable?
If not, where should I modify?

**Comment by sshaoshuai** [comment]
Of course, it is applicable since PointRCNN only needs the lidar data as input. 
For the pedestrian and cyclist training, currently I have not updated the config yet. You need to update the class name, mean size and FG thresh for each class, and you could also change the bin size and scope to fit with the corresponding class to get better results.

**Comment by diegocervera** [comment]
Does it not use the camera image for [sub-sampling](https://github.com/sshaoshuai/PointRCNN/blob/c83c4746f82addb4976cf2956c94923bd54cb816/lib/datasets/kitti_rcnn_dataset.py#L254) the points that go into the network?

**Comment by yewsiang** [comment]
Hi @sshaoshuai, may I know the FG thresh that was used for Pedestrian and Cyclist classes? Thank you very much!

**Comment by brudermueller** [comment]
@sshaoshuai May I ask again if this is actually the case that we do not need to adjust the bin-based loss if we just use LiDAR data as input and do not transform it into the camera coordinate system? As far as I see you are applying the bin-based loss in the x-z-plane corresponding to the BEV setting of the camera coordinate system where as it should correspond to the x-y-plane in the velodyne coordinate system. Do you have any recommendations how to adapt it? I tried adapting the loss myself, but changing the entire architecture to the Velodyne coordinate system seems very error prone. Also in your new toolbox you are not applying the bin-based loss for the center localization anymore. Could we potentially replace it by the L1-smooth loss?

**Comment by sshaoshuai** [comment]
@brudermueller , I think you could safely replace it with L1-smooth loss as shown in the OpenPCDet repo. I didn't try to use the bin-based loss in the implementation of PointRCNN of OpenPCDet since I just found the L1-smooth loss could already achieve promise results in the new OpenPCDet codebase, of course you could also try to use the bin-based loss in the OpenPCdet to compare the results with L1 loss in the OpenPCDet codebase.

**Comment by brudermueller** [comment]
@sshaoshuai Thanks for the quick reply! I appreciate your help! I have already been working a lot with your architecture in this repo now, even before you published the new toolbox. Would you still recommend to shift to your new toolbox instead? Further, do you have more findings and results with regard to pedestrians recently? I am mainly working in the area of crowd-navigation and hence mainly focus on the pedestrian class with my own LiDAR dataset.

**Comment by sshaoshuai** [comment]
OpenPCDet supports more models and datasets and new features than PointRCNN repo, also the performance is stronger, while this PointRCNN repo is more clear and simple. I think it depends on yourself and the progress of your project.  
Previously I mainly focus on the Vehicle detection and do not pay more attention on the pedestrian detection.

**Comment by BayaCherif** [comment]
> I use lidar data only to train PointRcnn, is the bin-based loss of axis xyz still applicable? If not, where should I modify?
> 
> In addition, how to set the params for pedestrian and cyclist training? I didn't see any explanation about it. Thanks.

Hi, can you please tell us if you got good results with training on ur custom dataset ?
